### PR TITLE
Add ACKs support to file read operation

### PR DESCRIPTION
**Pull request by skambalin**
- Added new parameters to low level `FileApi` and `CnsApi`
- Changed signature type to `ed25519` by default
- Added optional ACKs support (enabled by default for high-level API) to file API read operation
- Added tests to cover cases with enable and disable ACKs

### CODE REVIEW

**Comment by skambalin** [comment]
We need to wait for this issue to be resolved before merge:
https://github.com/Cerebellum-Network/ddc-storage-node/issues/355

File: packages/ddc/src/CnsApi/CnsApi.ts [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
+
+    if (!signer) {
+      throw new Error('Unnable to store CNS record. Signer required!');
     }
 
-    const signature = {
-      ...record.signature,
-      algorithm: record.signature.algorithm === 'ed25519' ? SigAlg.ED_25519 : SigAlg.SR_25519,
-    };
+    const signature = await createSignature(signer, createSignatureMessage(record));
```
Comment (ostafen) [inline packages/ddc/src/CnsApi/CnsApi.ts:43]: What is the `record` you are signing? The `PutRequest` or the `ActivityRecord`?

Comment (skambalin) [reply packages/ddc/src/CnsApi/CnsApi.ts:43]: @ostafen 
It is CNS API - there is no `ActivityRecord`. I sign CNS record here.

File: tests/specs/DdcApis.spec.ts [hunk 2]
```diff
@@ -26,33 +25,21 @@ const transports = [
   },
 ];
 
-describe.each(transports)('DDC APIs ($name)', ({ transport }) => {
-  let signer: Signer;
+const fileSpecVariants = [
+  { name: 'with ACKs', enableAcks: true },
+  { name: 'without ACKs', enableAcks: true },
```
Comment (upalinski) [inline tests/specs/DdcApis.spec.ts:30]: in both variants you have enableAcks: true

**Pull request merged**